### PR TITLE
added get_text and get_char_count methods to superdesk.etree

### PR DESCRIPTION
**Pull request by jerome-poisson**
needed for SDNTB-310

### CODE REVIEW

File: superdesk/etree.py [hunk 1]
```diff
@@ -21,14 +21,26 @@ def get_text_word_count(text):
     return len(text.split())
 
 
+def get_text(html):
+    try:
```
Comment (mdhaman) [inline superdesk/etree.py]: missing docstring.

**Comment by marwoodandrew** [comment]
Whilst not directly related to this pull request, I noticed the other day that the following test would fail, 

        self.assertEqual(2, get_word_count('<p>one</p><p>two</p>'))

I couldn't think of an elegant solution, other than perhaps injecting spaces into the string before parsing it, so I courageously ignored it.

File: superdesk/etree.py (hunk 1 above)
Comment (jerome-poisson) [inline superdesk/etree.py]: added, thanks for the review

**Comment by jerome-poisson** [comment]
@marwoodandrew thanks for the review.

The issue you're mentioning would need a separate ticket in my opinion.

I think it could be fixable with a parser with a list of block elements, adding extra space when it find one, that's doable without too much complications.

**Comment by jerome-poisson** [comment]
@marwoodandrew I've opened https://dev.sourcefabric.org/browse/SDESK-240 for the issue you mentioned. I'm merging the current P.R., thanks you all for review.

**Pull request merged**